### PR TITLE
feat: revamp the order history page

### PR DESCRIPTION
**Pull request by ardiankristo**
QR:
Screenshot from 2023-01-12 17-13-56
Screenshot from 2023-01-12 17-15-15
Screenshot from 2023-01-12 17-16-53
Screenshot from 2023-01-12 17-17-32

Retail / paylater:
Screenshot from 2023-01-12 17-18-23
Screenshot from 2023-01-12 17-18-39
Screenshot from 2023-01-12 17-23-42
Screenshot from 2023-01-12 17-23-50

VA (copy code):
Screenshot from 2023-01-12 17-19-06
Screenshot from 2023-01-12 17-19-24
Screenshot from 2023-01-12 17-19-30
Screenshot from 2023-01-12 17-19-37

E-wallet (link to payment):
Screenshot from 2023-01-12 17-20-02
Screenshot from 2023-01-12 17-20-07

Bank Transfer:
Screenshot from 2023-01-12 17-20-19
Screenshot from 2023-01-12 17-21-06
Screenshot from 2023-01-12 17-21-14
Screenshot from 2023-01-12 17-21-24

### CODE REVIEW

File: public/scss/pages/Account.module.scss [hunk 1]
```diff
@@ -1152,4 +1154,225 @@
...
+    @include flex(column, left);
+
+    gap: 4px;
+  }
+
+  &_paymentMethodDetailCard
+  {
+    margin: 4px 0;
+
+    & > div:nth-child(3) {
```
Comment (mayun19) [inline public/scss/pages/Account.module.scss:1174]: bracket nya di seragamkan enter kebawah

```
    & > div:nth-child(3) 
    {
      width: 180px !important;
    }
```

File: public/scss/pages/Account.module.scss [hunk 2]
```diff
@@ -1152,4 +1154,225 @@
...
+
+    border-right: 0.5px solid $color_dark_white_border;
+    border-bottom: 0.5px solid $color_dark_white_border;
+    border-left: 0.5px solid $color_dark_white_border;
+    border-bottom-right-radius: 4px;
+    border-bottom-left-radius: 4px;
+
+    & > svg {
+      margin-top: 1px;
+      height: 10px;
```
Comment (mayun19) [inline public/scss/pages/Account.module.scss:1221]: ini bisa pakai mixin

File: public/scss/pages/Account.module.scss [hunk 3]
```diff
@@ -1152,4 +1154,225 @@
...
+    padding: 8px;
+    gap: 4px;
+
+    border-right: 0.5px solid $color_dark_white_border;
+    border-bottom: 0.5px solid $color_dark_white_border;
+    border-left: 0.5px solid $color_dark_white_border;
+    border-bottom-right-radius: 4px;
+    border-bottom-left-radius: 4px;
+
+    & > svg {
```
Comment (mayun19) [inline public/scss/pages/Account.module.scss:1219]:
```
& > svg 
{
  ...
  ...
}
```

File: public/scss/pages/Account.module.scss [hunk 4]
```diff
@@ -1152,4 +1154,225 @@
...
+
+      &:last-child {
+        border-bottom: none;
+      }
+    }
+  }
+
+  &_paymentMethodDetailBankInfo
+  {
+    & > span {
```
Comment (mayun19) [inline public/scss/pages/Account.module.scss:1327]: ini juga

File: public/scss/pages/Account.module.scss [hunk 5]
```diff
@@ -1152,4 +1154,225 @@
...
+      @include typographyBuilder(400, 12, 16);
+
+      color: $color_black_text;
+    }
+
+    & > p {
```
Comment (mayun19) [inline public/scss/pages/Account.module.scss:1333]: idem {} nya

**Review by mayun19 — CHANGES_REQUESTED** [review]

**Comment by ardiankristo** [comment]
Screenshot from 2023-01-13 11-19-48
Screenshot from 2023-01-13 11-23-05

**Review by mayun19 — APPROVED** [review]

**Review by virginiaritonga — APPROVED** [review]

**Review by feliciakri — APPROVED** [review]

**Pull request merged**